### PR TITLE
Fixes spellcheck false positives and links

### PR DESCRIPTION
**Pull request by danimtb**
There were some broken or redirected links in the CI as well as some unknown words in the spell checker. This PR should fix all of them

### CODE REVIEW

File: spelling_wordlist.txt [hunk 1]
```diff
@@ -319,12 +366,15 @@ virtualrunenv
...
 watchOS
+weren
 werror
 wildcard
 wildcards
 workflow
 workflows
 Workflows
+workspaces
+Worspaces
```
Comment (memsharded) [inline spelling_wordlist.txt]: wrong

File: spelling_wordlist.txt [hunk 2]
```diff
@@ -319,12 +366,15 @@ virtualrunenv
 Virtualrunenv
 vswhere
 watchOS
+weren
```
Comment (memsharded) [inline spelling_wordlist.txt]: weren??

File: spelling_wordlist.txt [hunk 3]
```diff
@@ -1,11 +1,14 @@
...
 Admin
+apache
 api
 app
 apps
 appveyor
 Appveyor
+archiver
 archs
+aren
```
Comment (memsharded) [inline spelling_wordlist.txt]: aren?

File: spelling_wordlist.txt (hunk 2 above)
Comment (danimtb) [reply spelling_wordlist.txt]: Words like `weren't` are not well recognized

**Pull request merged**